dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -737,6 +737,15 @@
 			#io-channel-cells = <1>;
 		};
 
+		mailbox: mailbox@58026400 {
+			compatible = "st,stm32-hsem-mailbox";
+			reg = <0x58026400 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB4 0x02000000>;
+			interrupts = <125 0>;
```

Comment:
As interrupt 125 is valid for Cortex M7 only, could it be moved to _m7 specific file ?

Reply:
OK, I will try to move it to _m7, same like _m4.